snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/rule_mrcm_constraint.groovy

```diff
@@ -29,12 +29,32 @@ RevisionSearcher searcher = ctx.service(RevisionSearcher.class);
 Set<ComponentIdentifier> issues = Sets.newHashSet();
 
 final String allPrecoordinatedContent = "723594008";
```

Comment:
Please change this SCTID so that it points to `SnomedConstants.Concepts`.

Reply:
...by which I mean "remove the variable and use the constant directly" 😄

Reply:
Done